content/docs/reference-events.md

```diff
@@ -52,23 +51,19 @@ function onClick(event) {
     console.log(eventType); // => "click"
   }, 0);
 
-  // Won't work. this.state.clickEvent will only contain null values.
+  // کار نخواهد کرد. this.state.clickEvent فقط حاوی متغیرهای تهی خواهد بود.
   this.setState({clickEvent: event});
 
-  // You can still export event properties.
+  // همچنان قادر خواهید بود که دارایی‌های رویداد را صادر کنید
   this.setState({eventType: event.type});
 }
 ```
 
-> Note:
+> توجه:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
-
-## Supported Events {#supported-events}
+> اگر می‌خواهید که به دارایی‌های رویداد، به طور غیرهمزمان دسترسی داشته باشید باید `event.persist()` را در رویداد فراخوانی کنید. این موجب خواهد شد که رویداد سینتاتیک از روند برهم گذاری خارج شود و ارجاعات داده شده به آن رویداد در کد کاربر حفظ شود.
 
-React normalizes events so that they have consistent properties across different browsers.
-
-The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append `Capture` to the event name; for example, instead of using `onClick`, you would use `onClickCapture` to handle the click event in the capture phase.
+## رویدادهای پشتیبانی شده {#supported-events}
 
```

Comment:
ترجمه بخش زیر جا افتاده است:

React normalizes events so that they have consistent properties across different browsers.

The event handlers below are triggered by an event in the bubbling phase. To register an event handler for the capture phase, append Capture to the event name; for example, instead of using onClick, you would use onClickCapture to handle the click event in the capture phase.